### PR TITLE
Add /kill chat command

### PR DESCRIPTION
**Pull request by zaoqi**

### CODE REVIEW

**Comment by rubenwardy** [comment]
Please can you merge this and killme.

By that I mean make a mod with both kill and killme

**Comment by octacian** [comment]
Looks good, but could `/suicide` be changed to `/killme`? It seems a little, IDK, harsh.

File: mods/kill/init.lua [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	end
+end
+
+local function aliases(name, from)
+	if minetest.chatcommands[from] then
+		minetest.register_chatcommand(name, minetest.chatcommands[from])
+	end
+end
+
+minetest.register_chatcommand("suicide", {
```
Comment (rubenwardy) [inline mods/kill/init.lua]: please keep this as killme

File: mods/kill/init.lua [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	description = "Kill yourself to respawn",
+	func = function(name)
+		return kill.kill(minetest.get_player_by_name(name))
+	end
+})
+
+minetest.register_privilege("kill", {description = "Can kill the players", give_to_singleplayer = false})
```
Comment (rubenwardy) [inline mods/kill/init.lua]: > Can use the kill command to kill other players

**Comment by paramat** [comment]
I don't think we need another priv and the ability to kill other players. Such an admin ability would be better in builtim perhaps?
:-1:

**Comment by stujones11** [comment]
I think this can be a useful feature for server admins and has been included in the 3d_armor mod for some time on the request of others. However, I agree that this should be included in the builtin scripts, not MTG or armor.

**Comment by Wuzzy2** [comment]
I agree with paramat, `/kill` is something which should go straight into `builtin`.

**Comment by paramat** [comment]
Replaced by https://github.com/minetest/minetest/pull/5545 in the engine.

**Pull request closed without merging**